app/controllers/objects_controller.rb

```diff
@@ -62,21 +62,19 @@ def checksum
   # note: this is deliberately allowed to be a POST to allow for a large number of druids to be passed in
   # GET OR POST /v1/objects/checksums?druids[]=druid1&druids[]=druid2&druids[]=druid3
   def checksums
-    checksum_list, missing_druids, errored_druids = generate_checksum_list
-
-    bad_recs_msg = "\nStorage object(s) not found for #{missing_druids.join(', ')}" if missing_druids.any?
-    bad_recs_msg = (bad_recs_msg || '') + "\nProblems generating checksums for #{errored_druids.join(', ')}" if errored_druids.any?
+    checksum_list, missing_druids_list, errored_druids = generate_checksum_list
+    bad_recs_msg = "\nProblems generating checksums for #{errored_druids.join(', ')}" if errored_druids.any?
     if bad_recs_msg.present?
       render(plain: "409 Conflict - #{bad_recs_msg}", status: :conflict)
```

Comment:
i think this might more appropriately be a 500 now that we really don't expect this to happen (since missing druid was our only known error condition, and iirc the reason we chose 409, despite it not being a perfect fit semantically)